.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Thank you for your interest in ESLint! Please fill out the appropriate template for your issue below, as this will help us greatly in addressing your issue:
+
+1. Bug Reports
+2. New Rules
+3. Change Requests
+4. For any other questions, please stop by our chatroom: https://gitter.im/eslint/eslint
+
+Note that issues submitted with incomplete information may be closed without comment.
```

Comment:
This sounds a little harsh to me, are we really planning to just close incomplete issues?  My guess is no, we will at least give the submitter a chance to fill in more info before closing.  Can anyone think of a somewhat gentler warning?  I can't seem to think of an alternative.